src/main/java/mate/academy/model/MovieSession.java

```diff
@@ -0,0 +1,78 @@
+package mate.academy.model;
+
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.JoinColumn;
+import jakarta.persistence.JoinTable;
+import jakarta.persistence.ManyToMany;
+import jakarta.persistence.ManyToOne;
+import jakarta.persistence.Table;
+import java.time.LocalDateTime;
+import mate.academy.lib.Inject;
+
+@Entity
+@Inject
+@Table(name = "movie sessions")
+public class MovieSession {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+
+    @ManyToOne
+    @JoinColumn(name = "movie_id")
+    private Movie movie;
+
+    @ManyToMany
+    @JoinTable(name = "movieServices_cinemaHalls",
+            joinColumns = @JoinColumn(name = "movieService"),
+            inverseJoinColumns = @JoinColumn(name = "cinemaHall"))
+    private CinemaHall cinemaHall;

```

Comment:
The relationship between `MovieSession` and `CinemaHall` should be `@ManyToOne`, not `@ManyToMany`. Also, you don't need the `@JoinTable` annotation here. Replace it with `@JoinColumn(name = 'cinema_hall_id')`.